src/core/Wyam.Configuration/NuGet/PackageInstaller.cs

```diff
@@ -230,7 +230,7 @@ private void InstallPackages(NuGetPackageManager packageManager, IReadOnlyList<S
         {
             foreach (Package package in _packages.Values)
             {
-                package.Install(installationRepositories, installedPackages, packageManager).Wait();
+                package.Install(installationRepositories, installedPackages, packageManager).GetAwaiter().GetResult();
```

Comment:
I know, unlike `Task.Result` this does not produce AggregateExceptions (as far as I know), so less nested exceptions. But the documentation states 

>[This API supports the product infrastructure and is not intended to be used directly from your code. [...] This type and its members are intended for use by the compiler.](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.taskawaiter.getresult?view=netframework-4.8#System_Runtime_CompilerServices_TaskAwaiter_GetResult)

Not that I against it. But it should be an informed decision.

Reply:
I agree with the concern, thanks for calling it out. Normally I'd think carefully about using an internal API, but since the NuGet stuff is totally coming out in the next major version I'm just going to go ahead and YOLO merge this for now.